run.mac

```diff
@@ -43,19 +74,14 @@
 
 # If you want to see the list of all physical volume names (e.g. to set volume
 # IDs above) uncomment this line
-/g4simple/listPhysVols
+#/g4simple/listPhysVols
 
 # Set up gps after /run/initialize
 # This is for 208Tl events distributed throughout the "source" volume
-#/gps/particle ion
-#/gps/ion 81 208 0 0
-#/gps/energy 0 MeV
-#/gps/pos/shape Sphere
-#/gps/pos/radius 2 mm
-#/gps/pos/centre 0 0 5 cm
-#/gps/pos/confine source_PV
+#/gps/List
 /gps/particle opticalphoton
-/gps/polarization 1 0 0
+/gps/energy 9.68626 eV
+/gps/direction 1 0 0
+/gps/pos/centre 69 0 0
```

Comment:
<Ich kommentiers hier weil nur Github-Entenscheiß-Nutzer Issues anlegen können>
Ich würde vorschlagen, wir verlagern die Primarys in den Sourcecode, weil wir dann flexibler sind (einfacher randomisieren und loopen (obwohl das im Macro zur Not auch ginge)).
Richtung ist noch zu randomisieren; Position (--> Voxel) noch durchzuwechseln (Grenzen & Steps in macro)

@patgo25 kannst du das machen? Ich würde dann nachdem der Loop steht, die Zählwerte einzelner Voxel in ein Histogramm schreiben.

Das sollte dann der letzte größere Block sein.